File: 240424/팰린드롬 수 찾기/find-the-number-of-palindrome.py
X, Y = map(int, input().split())
nums_lst = list(range(X, Y + 1))

def is_palindrome(num):
    if num == num[::-1]:
        return True
    else:
        return False

cnts = 0
for num in nums_lst:
    num_lst = list(map(int, str(num)))
    if is_palindrome(num_lst):
        cnts += 1

print(cnts)

File: 260317/팰린드롬 수 찾기/find-the-number-of-palindrome.cpp
#include <bits/stdc++.h>
using namespace std;

bool is_palindrome(const vector<int>& num) 
{
    return num == vector<int>(num.rbegin(), num.rend());
}

int main() 
{
    int X, Y;
    cin >> X >> Y;

    // Please write your code here.
    int cnts = 0;
    for (int num = X; num <= Y; num++) 
    {
        string s = to_string(num);
        vector<int> num_lst;
        for (char c : s) 
        {
            num_lst.push_back(c - '0');
        }

        if (is_palindrome(num_lst)) 
        {
            cnts += 1;
        }
    }
    cout << cnts;
    return 0;
}
